### PR TITLE
[RLlib] Add compute log likelihoods test for Torch Policies with custom `action_distribution_fn`

### PR DESCRIPTION
**Pull request by avnishn**
Builds on #25877 

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: rllib/policy/tests/test_compute_log_likelihoods.py [hunk 1]
```diff
@@ -216,9 +222,42 @@ def test_sac_discr(self):
 
         do_test_log_likelihood(sac.SAC, config, prev_a)
 
+    def test_cql_cont(self):
```
Comment (gjoliver) [inline rllib/policy/tests/test_compute_log_likelihoods.py:225]: name?

File: rllib/algorithms/crr/crr.py [hunk 2]
```diff
@@ -43,7 +42,7 @@ def __init__(self, algo_class=None):
         # __sphinx_doc_end__
         # fmt: on
         self.replay_buffer_config = {
-            "type": "ReplayBuffer",
+            "type": MultiAgentReplayBuffer,
```
Comment (gjoliver) [inline rllib/algorithms/crr/crr.py:45]: is this really related??

**Review by gjoliver — COMMENTED** [review]
nice! a couple of quick question.

File: rllib/policy/tests/test_compute_log_likelihoods.py (hunk 1 above)
Comment (sven1977) [reply rllib/policy/tests/test_compute_log_likelihoods.py:225]: Consistent with all the other names in this test:
"cont": continuous actions
"disc": discrete actions

File: rllib/algorithms/crr/crr.py (hunk 2 above)
Comment (sven1977) [reply rllib/algorithms/crr/crr.py:45]: Yeah, we should probably revert this one-line change. Doesn't seem to be related.

I do agree we should go as generic as possible for our algo default configs, however, not sure about the pure offline RL algos (like CRR). I'm assuming multi-agent RL doesn't really work too well for offline RL anyways due to the extreme env dynamics that would need proper exploration (not available offline) to work.

**Review by sven1977 — APPROVED** [review]
Thanks for the PR and added test case @avnishn ! Could you fix the 1 nit (replay buffer default type for CRR)? Then we can merge.
